conda/cli/install.py

```diff
@@ -201,7 +201,10 @@ def install(args, parser, command='install'):
     specs = []
     if args.file:
         for fpath in args.file:
-            specs.extend(common.specs_from_url(fpath, json=context.json))
+            try:
+                specs.extend(common.specs_from_url(fpath, json=context.json))
+            except:
```

Comment:
It would be good to try to catch a specific kind of exception here.  I don't know what exception(s) common.specs_from_url might raise, but you should be able to figure that out by looking at it, or by feeding it bad data and seeing what exception gets raised.